.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [18.x]
```

Comment:
We should be adding node 20.x -- the current revision, and next LTS ( https://github.com/nodejs/release#release-schedule )

Reply:
Sure! Hopefully nothing breaks 🤞